wunderio/next4drupal-project
# Nex-68: improved search page accessibility

[Ticket](https://wunder.atlassian.net/browse/NEX-68)

*Changes proposed in this PR:*
- Restructure search results facets, result heading levels and remove props giving console errors.

*How to test:*
1. Open the link, perform a search and check console.
2. Use voice over after to listen for any strange announcements.
3. Use Wave a11y extension to check for errors (Ignore broken aria ref in search form - comes with the component)

*Link to feature environment:*
[test link](https://feature-nex-68-searcha11y-next.next4drupal-project.dev.wdr.io/search)
